scripts/la_selva_trees.py

```diff
@@ -1,22 +1,30 @@
 #retriever
 from retriever.lib.templates import Script
 from retriever.lib.models import Table, Cleanup, correct_invalid_value
+from retriever import VERSION
+from pkg_resources import parse_version
 
 
 class main(Script):
     def __init__(self):
         Script.__init__(self,
-                        tables={'trees': Table('trees', cleanup=Cleanup(correct_invalid_value, nulls=[-999]))},
-                        name="Tree growth, mortality, physical condition - Clark, 2006",
-                        tags=['plants', 'time-series'],
+                        tables={'trees': Table('trees', cleanup=Cleanup(correct_invalid_value, missingValues=[-999]))},
```

Comment:
it would be nice if you clean up some of these files in this was 

```python
class main(Script):
    def __init__(self, **kwargs):
        Script.__init__(self, **kwargs)
        self.title="Gulf of Maine intertidal density/cover (Petraitis et al. 2008)",
        self.citation="Peter S. Petraitis, Harrison Liu, and Erika C. Rhile. 2008. Densities and cover data for intertidal organisms in the Gulf of Maine, USA, from 2003 to 2007. Ecology 89:588.",
        self.name="intertidal-abund-me",
        self.ref="https://figshare.com/collections/DENSITIES_AND_COVER_DATA_FOR_INTERTIDAL_ORGANISMS_IN_THE_GULF_OF_MAINE_USA_FROM_2003_TO_2007/3300200",
        self.description="The data set provides access to data on densities and percent cover in the 60 experimental plots from 2003 to 2007 and to update data from 1996 to 2002 that are already published in Ecological Archives.It includes densities of mussels, an herbivorous limpet, herbivorous snails, a predatory snail, a barnacle , and fucoid algae and percent cover by mussels, barnacles, fucoids, and other sessile organisms.",
        self.retriever_minimum_version='2.0.dev',
        self.version='1.4.0',
        self.urls={"main": "https://ndownloader.figshare.com/files/5600831"},
        self.tables={"main": Table("main", cleanup=Cleanup(correct_invalid_value, missingValues=[-999.9]))}

        if parse_version(VERSION) <= parse_version("2.0.0"):
            self.shortname = self.name
            self.name = self.title
            self.tags = self.keywords
            self.tables = {"main": Table("main", cleanup=Cleanup(correct_invalid_value, nulls=[-999.9]))}
```

especially the ones where you to clean up `cleanup=Cleanup(correct_invalid_value, nulls=[-999.9]))}`